docs/websites/nginx/nginx-php-fpm-debian-8.md

```diff
@@ -0,0 +1,165 @@
+High performance Nginx and PHP over Debian 8
+19th May 2015 by Javier Briz
+
+
+This document describes how to install and setup an nginx server with PHP-FPM.
+This stack, together with a MySQL server, is what is called a **LEMP** server. After following this article, your PHP code will be
+able to connect to a MySQL server, but the MySQL server installation itself will not be explained;
+there are already [lots of guides](https://www.linode.com/docs/databases/mysql/) on this topic. 
+
+#Installation Prerequisites
+First of all, follow the [Getting Started Guide](https://www.linode.com/docs/getting-started/).
+It is also usefull to have a Fully Qualified Domain Name to have a handy pointer to the server.
+
+#Packages installation
+In this step, we are installing all the necessary packages.
+Run the following as root to install nginx, PHP and MySQL connector:
+
+        apt-get update
+        apt-get install nginx php5-fpm php5-mysql
+
+Make sure nginx and PHP-FPM are running
+
+        ps aux|grep -e nginx -e php
```

Comment:
We can use `systemctl status nginx` and `systemctl status php-fpm` here. This is more of a discussion point however. If you believe ps aux is the better choice then by all means stick with that.